aepsych/models/variational_gp.py

```diff
@@ -115,3 +118,70 @@ def get_config_options(cls, config: Config, name: Optional[str] = None):
         if options["likelihood"] is None:
             options["likelihood"] = BernoulliLikelihood()
         return options
+
+
+class OrdinalGP(VariationalGP):
+    """
+    Convenience class for using a VariationalGP with an OrdinalLikelihood.
+    """
+
+    outcome_type = "ordinal"
+    classname = "OrdinalGP"
+
+    def predict(self, xgrid, probability_space=False):
+        with torch.no_grad():
+            post = self.posterior(xgrid)
+        fmean = post.mean.squeeze()
+        fvar = post.variance.squeeze()
+
+        if probability_space:
+            fsd = torch.sqrt(1 + fvar)
+            probs = torch.zeros(*fmean.size(), self.likelihood.n_levels)
+
+            probs[..., 0] = self.likelihood.link(
+                (self.likelihood.cutpoints[0] - fmean) / fsd
+            )
+
+            for i in range(1, self.likelihood.n_levels - 1):
+                probs[..., i] = self.likelihood.link(
+                    (self.likelihood.cutpoints[i] - fmean) / fsd
+                ) - self.likelihood.link(
+                    (self.likelihood.cutpoints[i - 1] - fmean) / fsd
+                )
+            probs[..., -1] = 1 - self.likelihood.link(
+                (self.likelihood.cutpoints[-1] - fmean) / fsd
+            )
+            return probs
+        else:
+            return fmean, fvar
+
+    @classmethod
+    def get_config_options(cls, config: Config, name: Optional[str] = None):
+        options = super().get_config_options(config)
+
+        if options["likelihood"] is None:
+            options["likelihood"] = OrdinalLikelihood(n_levels=5)
+
+        dim = get_dim(config)
+
+        if config.getobj(cls.classname, "mean_covar_factory", fallback=None) is None:
```

Comment:
`cls.__name__` would get you classname without hardcoding it above, I believe -- why do it this way?